clang-tools-extra/clang-tidy/modernize/UseStartsEndsWithCheck.cpp

```diff
@@ -30,6 +30,17 @@ struct NotLengthExprForStringNode {
                IntegerLiteralSizeNode->getValue().getZExtValue();
       }
 
+      if (const auto *DeclRefNode = Node.get<DeclRefExpr>()) {
```

Comment:
llvm code guideline wants to ignore `{}` for `if` / `for` with single statement in body.